pkg/actuators/machine/utils.go

```diff
@@ -365,3 +330,22 @@ func conditionFailed() awsproviderv1.AWSMachineProviderCondition {
 		Reason: awsproviderv1.MachineCreationFailed,
 	}
 }
+
+// validateMachine check the label that a machine must have to identify the cluster to which it belongs is present.
+func validateMachine(machine machinev1.Machine) error {
+	if machine.Labels[machinev1.MachineClusterIDLabel] == "" {
+		return machinecontroller.InvalidMachineConfiguration("%v: missing %q label", machine.GetName(), machinev1.MachineClusterIDLabel)
+	}
+
+	return nil
+}
+
+// getClusterID get cluster ID by machine.openshift.io/cluster-api-cluster label
+func getClusterID(machine *machinev1.Machine) (string, bool) {
+	clusterID, ok := machine.Labels[machinev1.MachineClusterIDLabel]
+	// NOTE: This block can be removed after the label renaming transition to machine.openshift.io
+	if !ok {
```

Comment:
This can go away in a follow up, no need to support `upstreamMachineClusterIDLabel` anymore. Can you add a TODO?

Reply:
👍